resources/migrations/001_update_migrations.yaml

```diff
@@ -6323,6 +6323,64 @@ databaseChangeLog:
             relativeToChangelogFile: true
       rollback: # no rollback necessary, we're not removing the columns yet
 
+  - changeSet:
+      id: v50.2024-04-12T16:00:00
+      author: filipesilva
+      comment: Create cloud migration
+      changes:
+        - createTable:
+            tableName: cloud_migration
+            remarks: Migrate to cloud directly from Metabase
+            columns:
+              - column:
+                  name: id
+                  remarks: Unique ID
+                  type: int
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: external_id
+                  remarks: Matching ID in Cloud for this migration
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
+                  name: upload_url
+                  remarks: URL where the backup will be uploaded to
+                  type: ${text.type}
+                  constraints:
+                    nullable: false
+              - column:
+                  name: state
+                  remarks: 'Current state of the migration: init, setup, dump, upload, done, error, cancelled'
+                  type: varchar(32)
+                  defaultValue: init
+                  constraints:
+                    nullable: false
+              - column:
+                  name: progress
+                  remarks: Number between 0 to 100 representing progress as a percentage
+                  type: int
+                  defaultValue: 0
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  remarks: Timestamp when the config was inserted
+                  type: ${timestamp_type}
+                  defaultValueComputed: current_timestamp
```

Comment:
I'd be careful with `defaultValueComputed`, IIRC this will also add a `ON UPDATE CURRENT_TIMESTAMP` on some MySQL version. To be safe I'd use `(derive :model/CloudMigration :hook/timestamped?)` to let toucan2 handle this.

Reply:
I have `(derive :hook/timestamped?)` in the model already. So I should just remove `defaultValueComputed` right?